app/components/Views/Send/index.js

```
@@ -266,10 +267,14 @@ class Send extends PureComponent {
   /**
    * Handle deeplink txMeta recipient
    */
-  handleNewTxMetaRecipient = (recipient) => {
+  handleNewTxMetaRecipient = async (recipient) => {
     let ensRecipient, to;
+
     if (isENS(recipient)) {
       ensRecipient = recipient;
+      if (!to) {
+        to = await doENSLookup(ensRecipient, this.props.network);
+      }
     } else if (recipient && recipient.toLowerCase().substr(0, 2) === '0x') {
       to = toChecksumAddress(recipient);
     }
```

We probably shouldn't convert an address to the checksum and throw an error

---

Not understanding very well why we should throw an error here. When we have an address instead of an ens on the deep link we do this checksum 

Should not be converted?

---

We use the checksum as a way of verification that the address is correct, by doing an auto-checksum it bypasses this safety check. Please check https://github.com/MetaMask/mobile-planning/issues/288 for further reference.

---

Done it